### PR TITLE
student statuses crud, student edit, make student checkbox, BigDecimal prices

### PR DESCRIPTION
**Pull request by Shonorov**

### CODE REVIEW

File: src/main/java/com/ewp/crm/models/Student.java [hunk 1]
```diff
@@ -23,13 +24,13 @@
     private Date nextPaymentDate;
```
Comment (SevostyanovGerman) [inline src/main/java/com/ewp/crm/models/Student.java]: joda time (DateTime)

File: src/main/java/com/ewp/crm/service/impl/StudentServiceImpl.java [hunk 2]
```diff
@@ -24,25 +27,39 @@ public StudentServiceImpl(StudentRepository studentRepository, StudentStatusRepo
     }
 
     @Value("${price.month}")
-    private Long PRICE;
+    private String PRICE;
+    @Value("${default.student.status}")
```
Comment (SevostyanovGerman) [inline src/main/java/com/ewp/crm/service/impl/StudentServiceImpl.java]: ентеры между полями

File: src/main/java/com/ewp/crm/service/impl/StudentServiceImpl.java [hunk 3]
```diff
@@ -24,25 +27,39 @@ public StudentServiceImpl(StudentRepository studentRepository, StudentStatusRepo
...
-            StudentStatus status = studentStatusRepository.save(new StudentStatus("Auto generated " + client.getStatus().getName()));
+        if (client.getStudent() == null && client.getStatus().isCreateStudent()) {
+            StudentStatus status = studentStatusRepository.getStudentStatusByStatus(DEFAULT_STATUS);
+            if (status == null) {
+                status = studentStatusRepository.save(new StudentStatus(DEFAULT_STATUS));
+            }
             DateTime currentDate = new DateTime();
             if(client.getStatus().getName().equals("trialLearnStatus")) {
-                result = new Student(client, currentDate.plusDays(3).toDate(), currentDate.plusDays(3).toDate(), PRICE, PRICE, 0L, status, "");
+                result = new Student(client, currentDate.plusDays(3).toDate(), currentDate.plusDays(3).toDate(), new BigDecimal(PRICE), new BigDecimal(PRICE), new BigDecimal(0.00), status, "");
```
Comment (SevostyanovGerman) [inline src/main/java/com/ewp/crm/service/impl/StudentServiceImpl.java]: toDate() убрать везде joda time

**Comment by Shonorov** [comment]
Changes made

**Pull request merged**